src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -311,6 +311,75 @@ the library you can do so as follows:
 });
 ```
 
+#### Enriching ASP.NET Core Request Metrics
+
+> [!IMPORTANT]
+> Only applicable for .NET 8 and newer.
+
+ASP.NET Core supports enriching request metrics using `IHttpMetricsTagsFeature`.
+This feature allows you to add custom tags to metrics like `http.server.request.duration`,
```

Comment:
> like `http.server.request.duration`

I am not able to parse this part. What do we meant by "like" here? Is this enrichment applied to this specific metric, or any other metric that are similar to this or something else?  What if user creates custom metrics, will they also get enriched?

Reply:
IHttpMetricsTagsFeature applies specifically to metrics generated by the ASP.NET Core, but does not apply to custom user-defined metrics automatically.